libsolidity/formal/SymbolicVariables.cpp

```diff
@@ -21,12 +21,13 @@
 #include <libsolidity/ast/AST.h>
 
 using namespace std;
-using namespace dev;
-using namespace dev::solidity::smt;
+using namespace solidity;
+using namespace solidity::frontend;
```

Comment:
Maybe not much sense in importing this (``dev::solidity`` wasn't imported before and due to conflicts most of it needs to be qualified below anyways). If you don't import this, you can probably remove some of the ``smt::`` qualifiers below.